compiler/AST/symbol.cpp

```diff
@@ -1768,6 +1770,15 @@ void FnSymbol::finalizeCopy(void) {
      */
     update_symbols(this, map);
 
+    // Replace vararg formal if appropriate.
+    if (this->varargOldFormal) {
+      substituteVarargTupleRefs(this->body, this->varargNewFormals.size(),
```

Comment:
The general rule of thumb, which is easily applied in this example, is to place all of the actuals on a single
line or places them on individual lines.